rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/ast/statements.kt

```diff
@@ -1226,6 +1226,10 @@ data class CallStmt(
                             }
                         }
                 } catch (e: Exception) {
+                    MainExecutionContext
+                        .getMemorySliceMgr()
+                        ?.remove(MemorySliceId(MainExecutionContext.getConfiguration().defaultActivationGroupName, programToCall))
```

Comment:
This solution is little bit weak because it relies on a default active group name.